.azure/pipelines/signalr-daily-tests.yml

```diff
@@ -16,5 +16,5 @@ jobs:
     jobDisplayName: "SignalR Daily Tests"
     artifacts:
     - name: Windows_Logs
-      path: ../../artifacts/log/
+      path: ./artifacts/log/
```

Comment:
Change to `artifacts/log/` to match other instances in the repo: https://github.com/dotnet/aspnetcore/blob/d4998e0e4a1407f7a53022371792c3cb75fb8e21/.azure/pipelines/ci.yml#L107

Reply:
Got it -- done!